### ISSUE TITLE
No additional cost

### ISSUE THREAD

**Issue by dt98** [MicrosoftDocs/SCCMdocs#2161]
Just want some clarification on the part about no additional cost, I feel it is a little misleading just based on the way it's positioned.  Correct me if I'm wrong, but don't you need an Azure subscription (which is a cost item) along with one of the E3/E5 licenses that you have listed in this article (which is also a cost) in order to get Desktop Analytics working?  By having the bullet points of the E3/E5 licenses right beside the paragraph about "no additional cost", it makes it look like you only need E3/E5 licensing.  Maybe it should say something along the lines of "Beyond the cost of these license subscriptions along with an Azure subscription."  Thanks for hard work on these articles!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2d8d7ece-b19f-1431-a619-e57ce6469c28
* Version Independent ID: 86a3d7a5-695b-d3d9-498c-e6b6e03169f8
* Content: [Desktop Analytics - Configuration Manager](https://docs.microsoft.com/en-us/configmgr/desktop-analytics/overview#feedback)
* Content Source: [sccm/desktop-analytics/overview.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/desktop-analytics/overview.md)
* Product: **configuration-manager**
* Technology: **configmgr-other**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks @dt98 for the feedback! I'm checking on this.

**Comment by dt98** [comment]
Any update on this?  I know you're busy, but just thought I'd ask.

**Comment by aczechowski** [comment]
No, sorry. Still trying to get a response

**Comment by aczechowski** [comment]
I have a pull request in to clarify things a bit. This issue will auto close when it merges. Then it should publish live tomorrow. 

We already mention "an active global Azure subscription" as a *technical* prereq, but I added it as an explicit licensing prereq as well. I also added a note that most of the equivalent subscriptions for Configuration Manager include Azure AD. For example, EMS and M365. So it shouldn't be an additional cost beyond what you already have. The blurb to which you're referring is specifically about Log Analytics, so I tried to separate that a bit in my edit. Take a look tomorrow and let me know if you have any further feedback.
Thanks again!

p.s. while not tested, theoretically it might be technically possible to do the DA scenarios with an eval of ConfigMgr and an Azure trial or pay-as-you-go subscription. But that wouldn't be sustainable or scalable, only for eval/demo purposes (if even possible). I didn't include that in the article revision, just FYI 😄
